src/components/FilterButton/FilterButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import cn from 'classnames';
+import { FilterOptions } from '../../types/FilterOptions';
+
+const { All, Active, Completed } = FilterOptions;
+
+const getFilterOption = (filter: string) => {
```

Comment:
do you need this helper? Seems you can just change the filter state and that's it - you already have an enum for that